src/main/java/org/skriptlang/skript/lang/script/Script.java

```diff
@@ -24,11 +24,11 @@
 import org.jetbrains.annotations.Unmodifiable;
 import org.skriptlang.skript.lang.structure.Structure;
 
-import java.util.Collections;
-import java.util.HashSet;
 import java.util.List;
```

Comment:
```suggestion
import java.util.Collections;
import java.util.HashSet;
import java.util.List;
import java.util.Map;
```
to remove the diff (pt1)